Bar component enhancement

michac789 opened the pull request.

Commit:
Bar enhancement: add back button, modify style, custom spacing

Commit:
Bar updated snaphot test case

Commit:
Bar minor story name change, add one visual test

chiahou commented on src/@next/Bar/BarStyle.ts:
```diff
@@ -2,14 +2,20 @@ import styled from 'styled-components';
+  min-height: ${props => (props.smallScreen ? '70' : '90')}px;
+  height: fit-content;
 
   @media (max-width: ${Breakpoints.large}) {
```
@michac789 Given that we are using min-height here, I believe we should remove the height attribute to prevent content overflow.

Commit:
Bar fix content overflowing on small screen

chiahou reviewed (approved):
qc passed, thanks!

Commit:
Bar added customActionGroup, modify prev commit based on feedback

Commit:
Bar added visual test cases

ninariccimarie commented on src/@next/Bar/Bar.stories.tsx:
```diff
@@ -13,7 +13,10 @@ export default {
   decorators: [
     Story => (
-      <BaseContainer style={{ height: '200px' }}>{Story()}</BaseContainer>
+      <>
```
i think you can use instead: 
`[withGlintsPortalContainer]`

michac789 replied:
@ninariccimarie 

edit:
This works:
```
  decorators: [
    Story => (
      <>
        <BaseContainer style={{ height: '200px' }}>{Story()}</BaseContainer>
      </>
    ),
    withGlintsPortalContainer,
  ],
```


~~using that decorator only caused this issue:~~
![image](https://github.com/glints-dev/glints-aries/assets/67429929/7411ee66-6330-4ad7-b4b5-069ec1ace019)

~~currently it is like this:~~
![image](https://github.com/glints-dev/glints-aries/assets/67429929/669c1d39-8ba4-446f-a6b9-dacd43708fc5)
~~(I need to put the customStyle height of 200px)~~

michac789 replied:
@ninariccimarie ~~do you want me to modify the decorator so it can take in an argument of customStyle to be passed on to the div instead, or just leave it as it is right now (but I don't use that decorator)?~~

michac789 replied:
will settle with this for now, lmk if u have anything else

ninariccimarie commented:
not really sure about the diff for the other snapshots that were updated. when swiping, there doesn't seem to be any difference? 🤔 can you help to verify this @michac789

michac789 commented:
> not really sure about the diff for the other snapshots that were updated. when swiping, there doesn't seem to be any difference? 🤔 can you help to verify this @michac789

preety sure I changed the spacings a bit based on the Figma (new), so I expected minor change (just a slight space) but yeah tbh not really visible hmm, let me check real quick and will update you again


Edit:
@ninariccimarie 
So it turns out yeah the snapshots are exactly the same 🤣 
Thanks for pointing that out 😄 
On small screen, there is a subtle difference. On large screen however, if the content is short there is no difference at all. What was being modified on large screen was the padding on the vertical axis, however due to the minimum height property it does not affect anything since the content is short.

*Edit2: I reverted that commit, is it okay? If not okay perhaps I can rebase then pick the commit (not sure if it will work but I can try). or worst case I can create a new branch and commit everything again*

Commit:
Revert "Bar updated snaphot test case"

This reverts commit 04108a1f9b43c57996b78f04876def4d02cb178f.

Commit:
Bar minor code cleanup change

ninariccimarie reviewed (approved):
LGTM 👍 thanks
